course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -16,7 +16,7 @@
     "## Iris introduction course\n",
     "# 6. Data Processing\n",
     "\n",
-    "**Learning Outcome**: by the end of this section, you will be able to apply arithmetic and statistical operations on cube data.\n",
+    "**Learning Outcome**: by the end of this section, you will be able to apply arithmetic and statistical operations on the data of a Cube.\n",
```

Comment:
I think "the data of a Cube" is still not quite right.
( after all, we definitely *don't* mean `cube.data` )
Maybe just "perform arithmetic and statistical operations on a Cube" ?

Reply:
Yeah, I guess "the data of a Cube" misses out the importance of metadata handling.
I will go with your suggestion